libs/deal-ticket/src/hooks/use-fee-deal-ticket-details.tsx

```diff
@@ -1,5 +1,9 @@
 import { FeesBreakdown } from '@vegaprotocol/markets';
```

Comment:
**chore (non-blocking)**: I'd like to see more tests for this hook. Initially I thought `formatValue` and formatRange`  were untested, but @MadalinaRaicu pointed me to the tests for the new `addDecimalsFormatNumberQuantum` function that does a great job of encapsulating how that function works. 

I still want to see more hook tests, but the `formatValue` function has enough branches that some tests would be good to show a few paths through the code. Decimal formatting has been error prone in the past, so the more tests at all levels, the better off we'll be.

Reply:
I'm gonna add some while this PR is still in draft